heartbeat/monitors/monitor.go

```diff
@@ -217,14 +215,18 @@ func (m *Monitor) Start() {
 	}
 
 	m.stats.StartMonitor(int64(m.endpoints))
+	m.state = MON_STARTED
 }
 
-// Stop stops the Monitor's execution in its configured scheduler.
-// This is safe to call even if the Monitor was never started.
+// Stop stops the monitor without freeing it in global dedup
```

Comment:
```suggestion
// Stop the monitor without freeing it in global dedup
```
I know the function name is a proper noun here but it feels kinda redundant.